.travis.yml

```diff
@@ -4,6 +4,14 @@ node_js:
 - '4'
 - '6'
 
+# Run on travis with both yarn enabled and disabled.
+env:
+- YARNPKG=true
+- YARNPKG=false
+before_install:
+- if [[ $YARNPKG != "true" ]]; then rm yarn.lock; fi
```

Comment:
doesn't this need an else branch that does `git checkout .` to restore the lock file? I don't know exactly how the matrix build runs but I think it multiplies `env` by each `node_js` version, right? That would mean Node 4 tests would get a yarn.lock file but Node 6 tests would not.